global/cta-analytics@1.0.js

```diff
@@ -0,0 +1,41 @@
+!function () {
```

Comment:
We'll add it separately in an embed so that it's clear that it is not gonna affect the nav.

Reply:
But will be on all pages?

Reply:
Yeah it will have to be on all pages because we need to add this so that current analytics tracking setup for other CTAs on pages doesn't break. 
I suppose, we can just add it to the nav symbol as a quick way of adding this script on all the pages. 
Otherwise, we could include it under Custom Code at the project level. In that case, we can defer the script.

Reply:
Custom code footer/header with async probably is needed. Otherwise it won't be evaluated in time for interaction